modules/aws/vpc.go

```diff
@@ -22,14 +22,18 @@ type Vpc struct {
 
 // Subnet is a subnet in an availability zone.
 type Subnet struct {
-	Id               string // The ID of the Subnet
-	AvailabilityZone string // The Availability Zone the subnet is in
+	Id               string            // The ID of the Subnet
+	AvailabilityZone string            // The Availability Zone the subnet is in
+	Tags             map[string]string // The tags associated with the subnet
 }
 
 var vpcIDFilterName = "vpc-id"
 var vpcResourceIdFilterName = "resource-id"
 var vpcResourceTypeFilterName = "resource-type"
 var vpcResourceTypeFilterValue = "vpc"
+var subnetResourceIdFilterName = "resource-id"
+var subnetResourceTypeFilterName = "resource-type"
+var subnetResourceTypeFilterValue = "subnet"
```

Comment:
I wasn't sure if this near-duplication is wanted or not? - I've implemented `subnet` tags similar to the way the `vpc` tags were done but don't foresee additional/other duplication of these sorts of things as there are only `vpc` and `subnet` within this `vpc` module.

Reply:
Two thoughts:
- Any reason all these vars should not be `const`s?
- Since `vpcResourceIdFilterName` and `subnetResourceIdFilterName` have identical values, we could have just one `resourceIdFilterName` instead. Same for `vpcResourceTypeFilterName`/`subnetResourceTypeFilterName`.

Reply:
I tend to agree - I’ll update the PR and consolidate these as suggested.